src/styles/main.scss

```diff
@@ -0,0 +1,131 @@
+@import "products", "cart", "login";
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Dosis", sans-serif;
```

Comment:
Don't apply the base font family with the reset. Use a separate `body` definition.